lib/Hoymiles/src/parser/AlarmLogParser.cpp

```diff
@@ -38,34 +38,34 @@ const std::array<const AlarmMessage_t, ALARM_MSG_COUNT> AlarmLogParser::_alarmMe
+    { AlarmMessageType_t::ALL, 48, "FB clamp overvoltage", "Überspannung der FB-Klemme", "" },
+    { AlarmMessageType_t::ALL, 49, "FB clamp overvoltage", "Überspannung der FB-Klemme", "" },
 
-    { AlarmMessageType_t::ALL, 61, "Calibration parameter error", "", "" },
-    { AlarmMessageType_t::ALL, 62, "System configuration parameter error", "", "" },
-    { AlarmMessageType_t::ALL, 63, "Abnormal power generation data", "", "" },
+    { AlarmMessageType_t::ALL, 61, "Calibration parameter error", "Kalibrierungsparameterfehler", "" },
+    { AlarmMessageType_t::ALL, 62, "System configuration parameter error", "Fehler bei den Systemkonfigurationsparametern", "" },
+    { AlarmMessageType_t::ALL, 63, "Abnormal power generation data", "Ungewöhnliche Daten zur Stromerzeugung", "" },
 
-    { AlarmMessageType_t::ALL, 71, "Grid overvoltage load reduction (VW) function enable", "", "" },
-    { AlarmMessageType_t::ALL, 72, "Power grid over-frequency load reduction (FW) function enable", "", "" },
-    { AlarmMessageType_t::ALL, 73, "Over-temperature load reduction (TW) function enable", "", "" },
+    { AlarmMessageType_t::ALL, 71, "Grid overvoltage load reduction (VW) function enable", "Aktivierung der Netzüberspannungslastreduzierungsfunktion (VW)", "" },
+    { AlarmMessageType_t::ALL, 72, "Power grid over-frequency load reduction (FW) function enable", "Aktivierung der Funktion zur Reduzierung der Überfrequenzlast (FW) im Stromnetz", "" },
+    { AlarmMessageType_t::ALL, 73, "Over-temperature load reduction (TW) function enable", "Aktivierung der Funktion zur Reduzierung der Übertemperaturlast (TW)", "" },
 
-    { AlarmMessageType_t::ALL, 95, "PV-1: Module in suspected shadow", "", "" },
-    { AlarmMessageType_t::ALL, 96, "PV-2: Module in suspected shadow", "", "" },
-    { AlarmMessageType_t::ALL, 97, "PV-3: Module in suspected shadow", "", "" },
-    { AlarmMessageType_t::ALL, 98, "PV-4: Module in suspected shadow", "", "" },
+    { AlarmMessageType_t::ALL, 95, "PV-1: Module in suspected shadow", "PV-1: Modul im vermuteten Schatten", "" },
+    { AlarmMessageType_t::ALL, 96, "PV-2: Module in suspected shadow", "PV-2: Modul im vermuteten Schatten", "" },
+    { AlarmMessageType_t::ALL, 97, "PV-3: Module in suspected shadow", "PV-3: Modul im vermuteten Schatten", "" },
+    { AlarmMessageType_t::ALL, 98, "PV-4: Module in suspected shadow", "PV-4: Modul im vermuteten Schatten", "" },
 
     { AlarmMessageType_t::ALL, 121, "Over temperature protection", "Übertemperaturschutz", "Protection antisurchauffe" },
-    { AlarmMessageType_t::ALL, 122, "Microinverter is suspected of being stolen", "", "" },
-    { AlarmMessageType_t::ALL, 123, "Locked by remote control", "", "" },
+    { AlarmMessageType_t::ALL, 122, "Microinverter is suspected of being stolen", "Verdacht auf Diebstahl des Mikrowechselrichters", "" },
+    { AlarmMessageType_t::ALL, 123, "Locked by remote control", "Per Fernbedienung sperrbar", "" },
```

Comment:
Shouldn't this rather be "Durch Fernbedienung gesperrt"?

Reply:
I think it's good and I changed it.